PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -46,6 +46,9 @@ class PurchasesTests: XCTestCase {
         receiptFetcher = MockReceiptFetcher(requestFetcher: requestFetcher, systemInfo: systemInfoAttribution)
         attributionFetcher = MockAttributionFetcher(attributionFactory: MockAttributionTypeFactory(),
                                                     systemInfo: systemInfoAttribution)
+        backend = MockBackend(httpClient: MockHTTPClient(systemInfo: systemInfo,
+                                                         eTagManager: MockETagManager()),
+                              apiKey: "mockAPIKey")
```

Comment:
I know the indentation is from previous commit, but let's update it

Reply:
I think this is correct, this parameter is for `MockBackend.init`. Or are you pointing to something else?

Reply:
Nope, you are right, sorry for the noise!

Reply:
No worries!